packages/pwa-kit-create-app/assets/pwa/default.js

```diff
@@ -41,6 +41,10 @@ module.exports = {
             siteId: '${einsteinApi.siteId}'
         }
     },
+    // Flag that marks this config for a live production site
+    // This is currently used to flag Einstein activities as either coming from a production 
+    // environment or not.
+    ssrIsProduction: false,
```

Comment:
Comment from slack:

"We can also opt to be more specific with our config, picking a setting name thats verbose like “send analytics to einstein as production” or something long enough to be clear." - @johnboxall 

I would agree with this comment. Although I don't think this config option is going to be permanent, I think it's good not to introduce confusion.